packages/terra-functional-testing/src/commands/validates/screenshot.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * The screenshot comparisons.  It will capture screenshots of a specified element
+ * and assert the screenshot comparison results are within the mismatch tolerance or are an exact match
+ *
+ * This should be used within an `it` block.
+ *
+ * @param {[name, options]} [args] - the list of test arguments to parse.
```

Comment:
nit:  Is this param doc intentional? I don't think I understand what it's defining here

Reply:
This doc is extra and not needed. Removed. https://github.com/cerner/terra-toolkit/commit/9933873194e2d7e6d395853f313cdd4cf89d1212